ui/test/dashboards/components/GraphOptionsCustomizableField.test.tsx

```diff
@@ -1,75 +0,0 @@
-import {shallow} from 'enzyme'
```

Comment:
We will probably need to add back in component tests for these components at some point